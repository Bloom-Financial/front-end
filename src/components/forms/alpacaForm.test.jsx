import React from 'react';
import { render } from '@testing-library/react';
import AlpacaForm from './AlpacaForm';


describe('alpaca form', () => {
  it('tests the alpaca form component', () => {
    const { asFragment } = render(<AlpacaForm/>);
    expect(asFragment()).toMatchSnapshot();
  });
});
