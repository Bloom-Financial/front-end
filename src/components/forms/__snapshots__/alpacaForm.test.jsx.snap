// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`alpaca form tests the alpaca form component 1`] = `
<DocumentFragment>
  <div>
    <form>
      <input
        placeholder="symbol"
        type="text"
        value=""
      />
      <input
        placeholder="quantity"
        type="number"
        value=""
      />
      <select>
        <option
          value="buy"
        >
          Buy
        </option>
        <option
          value="sell"
        >
          Sell
        </option>
      </select>
      <button>
        Submit
      </button>
    </form>
  </div>
</DocumentFragment>
`;

exports[`alpaca form tests the header component 1`] = `
<DocumentFragment>
  <div>
    <form>
      <input
        placeholder="symbol"
        type="text"
        value=""
      />
      <input
        placeholder="quantity"
        type="number"
        value=""
      />
      <select>
        <option
          value="buy"
        >
          Buy
        </option>
        <option
          value="sell"
        >
          Sell
        </option>
      </select>
      <button>
        Submit
      </button>
    </form>
  </div>
</DocumentFragment>
`;
